x-team/xp-ui
# Change wording in the application text

**Release Type:** *Non-Breaking Feature* 

Fixes https://x-team-internal.atlassian.net/browse/XP-3663

## Description

This PR contains the following changes for the Application text:
- "You've already applied..." for "You applied...."
- “Check your email for next steps.” for “We will reach out soon via email to talk about next steps.”

## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs



branch | PR
------ | ------
xp_registration_branch_name | [XP Registration](https://github.com/x-team/xp-registration/pull/170)

## Steps to Test or Reproduce

1. Run `npm run storybook:build` and open the project in your browser
2. Look at the `AplicantScreen` the following 2 states where should be reflected the new words
- With notification
- With menu links and notification
3. Look at the `JobApplicationCard` the following 2 states where should be reflected the new words
- Withdraw
- Withdraw + is saving
4. Look at the `JobsPageLayout` the following debug case where should be reflected the new words
- Job details page (applied)

## Impacted Areas in Application

- `/src/components/ui/AplicantScreen.stories.js`
- `/src/components/ui/JobApplicationCard.stories.js`
- `/src/components/ui/JobsPageLayout.stories.js`

## Screenshots

- `AplicantScreen`

![Screen Shot on 2020-03-30 at 15:00:09](https://user-images.githubusercontent.com/6010411/77952436-a9a5ce80-7299-11ea-8247-5f8e3a376599.png)
![Screen Shot on 2020-03-30 at 15:01:00](https://user-images.githubusercontent.com/6010411/77952444-ad395580-7299-11ea-9ad5-d439b8799910.png)

- `JobApplicationCard`

![Screen Shot on 2020-03-30 at 15:02:44](https://user-images.githubusercontent.com/6010411/77952469-b5919080-7299-11ea-930c-538b6854a2bd.png)
![Screen Shot on 2020-03-30 at 15:03:18](https://user-images.githubusercontent.com/6010411/77952475-b75b5400-7299-11ea-8624-7a7d01642430.png)

- `JobsPageLayout`

![Screen Shot on 2020-03-30 at 15:04:51](https://user-images.githubusercontent.com/6010411/77952484-bc200800-7299-11ea-80a7-4b40362b5e18.png)
